service_container.rst

```diff
@@ -133,68 +123,60 @@ the service container *how* to instantiate it:
 
         # app/config/services.yml
         services:
-            app.message_generator:
-                class:     AppBundle\Service\MessageGenerator
-                arguments: []
+            # default configuration for services in *this* file
+            _defaults:
+                autowire: true
+                autoconfigure: true
+
+            # load services from whatever directories you want (you can update this!)
+            AppBundle\:
+                resource: '../../src/AppBundle/{Service,EventDispatcher,Twig,Form}'
 
     .. code-block:: xml
 
         <!-- app/config/services.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd">
-
-            <services>
-                <service id="app.message_generator" class="AppBundle\Service\MessageGenerator">
-                </service>
-            </services>
-        </container>
+        TODO
 
     .. code-block:: php
 
         // app/config/services.php
-        use AppBundle\Service\MessageGenerator;
-        use Symfony\Component\DependencyInjection\Definition;
+        TODO
```

Comment:
Prototypes don't exist for php code so imo we should either remove this doc block or let the current one with a comment explaining that it's not possible to register entire folders as with other loaders.